ci-scripts/yocto.groovy

```diff
@@ -229,13 +229,25 @@ void buildManifest(String variantName, String imageName, String layerToReplace="
 
         } finally { // Archive cache even if there were errors.
             archiveCache(yoctoDir, archive, env.YOCTO_CACHE_ARCHIVE_PATH)
-            // If nightly build, we store the artifacts as well
+            // Check if we want to store the artifacts as well
+            boolean archiveArtifacts = false
             boolean nightly = env.NIGHTLY_BUILD == "true"
             boolean weekly = env.WEEKLY_BUILD == "true"
-            if (nightly || weekly) {
+            try {
+                archiveArtifacts = env.JOB_ARCHIVE_ARTIFACTS
```

Comment:
Changing the name for the archiveArtifacts would be nice. We have multiple uses of it.

Reply:
Done. Please review the name changes I did. Both for the boolean and the method